target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.h

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Realtek RTL83XX clock headers
+ * Copyright (C) 2022 Markus Stockhausen
+ */
+
+/*
+ * Several stuff
+ */
+
+#define RTL_SRAM_MARKER				(0x42C0FFEE)
+
+/*
+ * Switch registers
+ */
+
+#define RTL_SW_CORE_BASE			(0xBB000000)
+
+#define RTL838X_PLL_GLB_CTRL			(0x0FC0)
+#define RTL839X_PLL_GLB_CTRL			(0x0024)
+#define RTL838X_PLL_CPU_CTRL0			(0x0FC4)
+#define RTL839X_PLL_CPU_CTRL0			(0x0028)
+#define RTL838X_PLL_CPU_CTRL1			(0x0FC8)
+#define RTL839X_PLL_CPU_CTRL1			(0x002C)
+#define RTL838X_PLL_LXB_CTRL0			(0x0FD0)
+#define RTL839X_PLL_LXB_CTRL0			(0x0038)
+#define RTL838X_PLL_LXB_CTRL1			(0x0FD4)
+#define RTL839X_PLL_LXB_CTRL1			(0x003C)
+#define RTL838X_PLL_MEM_CTRL0			(0x0FDC)
+#define RTL839X_PLL_MEM_CTRL0			(0x0048)
+#define RTL838X_PLL_MEM_CTRL1			(0x0FE0)
+#define RTL839X_PLL_MEM_CTRL1			(0x004C)
```

Comment:
Now that these are in a separate file, it would make more sense to me to group these by family, instead of by type (GLB_CTRL, CPU_CTRL0, ...)

Reply:
fixed in next update